src/features/import/utils/prepareImportOperations.ts

```diff
@@ -109,6 +109,17 @@ export default function prepareImportOperations(
           });
         }
 
+        if (column.kind === ColumnKind.ENUM) {
+          column.mapping.forEach((mappedColumn) => {
+            if (mappedColumn.value === row.data[colIdx] && mappedColumn.key) {
+              personImportOps[rowIndex].data = {
+                ...personImportOps[rowIndex].data,
+                [`${column.field}`]: mappedColumn.key,
+              };
+            }
+          });
+        }
```

Comment:
Please add a test cases that encompasses this in `prepareImportOperations.spec.ts`.

Reply:
A test should make sure that mapping empty values work as expected. Right now, it does not seem to be working. Configuring empty values using the UI yields the expected configuration in the store, but when submitted to the backend the mappings for "empty values" seem lost.

![image](https://github.com/user-attachments/assets/dc798156-6dc6-4446-a9e0-a0669bdcabf7)

![image](https://github.com/user-attachments/assets/dcdf1f8d-d2a2-401f-ae87-b157ee861fba)

Reply:
Fixed. Also added `undefined` to `CellData` type in order to allow for testing of undefined cell values which is seen in real world usage.

Reply:
I haven't looked at the code yet, but I did just try an import using empty values and it works, so this is resolved. Nice work!